_data/contributors.yml

```diff
@@ -22,17 +25,35 @@
     - Just the beginning
     - Configuration Demystified
     - Ecto query composition
+    - 'Umbrellas: only when it rains?'
 
-- github: ybur-yug
-  name:
+- github: gemantzu
+  name: George Mantzouranis
   intro:
   homepage:
-  twitter:
+  twitter: gemantzu
+  contributions:
+    - translator
+    - blog
+  languages:
+    - gr
+  articles:
+    - Reviewing Elixir in Action, Second Edition
+    - Reviewing Functional Web Development with Elixir, OTP, and Phoenix
+
+- github: notactuallypagemcconnell
```

Comment:
@notactuallypagemcconnell I update some of your info.  If you want to submit a PR with your intro and others that'd be an easy hacktoberfest contribution 😁